pandas/io/pytables.py

```diff
@@ -1816,10 +1817,33 @@ class GenericIndexCol(IndexCol):
     def is_indexed(self):
         return False
 
-    def convert(self, values, nan_rep, encoding, errors):
-        """ set the values from this selection: take = take ownership """
+    def convert(self, values, nan_rep, encoding, errors, start=None,
+                stop=None):
+        """ set the values from this selection: take = take ownership
+
+        Parameters
+        ----------
+
+        values :
+            Expected to be passed but ignored in this implementation.
```

Comment:
can you write that these are, e.g.

values: np.ndarray
 nan_rep : str
encoding: str
errors : str (I think)